src/main/java/com/dynatrace/openkit/core/util/DefaultLogger.java

```diff
@@ -45,7 +45,7 @@ public DefaultLogger(boolean verbose) {
 
     @Override
     public void error(String message) {
-        System.out.println(getUTCTime() + " [ERROR] " + message);
+        System.out.println(getUTCTime() + " ERROR [" + Thread.currentThread().getName() + "] " + message);

```

Comment:
I'd suggest introducing a helper method.

In case the log output string is changed, it's not necessary to change it in N places

Reply:
As discussed, I still stick to the code duplication here (code still fits on one page).